dbld/builddeps

```diff
@@ -91,22 +91,20 @@ function download_target() {
 }
 
 function filter_packages_by_platform {
-    FILENAME=$1
```

Comment:
It's common practice to name positional arguments via assignments so that we don't need to remember which argument is which. So I would leave FILENAME here.

Reply:
ohh okey, I put it back the FILENAME